### PR TITLE
Alcail/added screen sharing control to VideoGallery story

### PR DESCRIPTION
**Pull request by alcail**
# What
Moved screen sharing experience in Storybook to VideoGallery story by adding a screen sharing option control

# Why
bug (https://skype.visualstudio.com/SPOOL/_workitems/edit/2641856)

# How Tested
ran Storybook locally 

image

- No sharing/default
  image

- local sharing
  image

- remote sharing
  image

# Process & policy checklist

- [x] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

**Review by prprabhu-ms — APPROVED** [review]

**Comment by alkwa-msft** [comment]
The Screen sharing options to me sounds like different ways to share my screen. I feel this is not what this knob is about. 

Do we call it something like ScreenSharing experience? Perhaps we can try to understand what we are trying to solve and to figure out a better name  so it is clear to storybook users what the are they really seeing?

**Comment by alcail** [comment]
> The Screen sharing options to me sounds like different ways to share my screen. I feel this is not what this knob is about.
> 
> Do we call it something like ScreenSharing experience? Perhaps we can try to understand what we are trying to solve and to figure out a better name so it is clear to storybook users what the are they really seeing?
> 
> Possible suggestion: Screenshare Viewing Experience : Presenter / Viewer / None

@alkwa-msft, change done

**Review by alkwa-msft — APPROVED** [review]

File: packages/storybook/stories/VideoGallery/VideoGallery.stories.tsx [hunk 1]
```diff
@@ -119,6 +120,25 @@ const VideoGalleryStory = (args): JSX.Element => {
...
 
+  const localParticipant = MockLocalParticipant;
+  localParticipant.isScreenSharingOn = args.screenShareExperience === 'presenter';
+
+  if (remoteParticipants.length > 0) {
+    remoteParticipants[0].isScreenSharingOn = args.screenShareExperience === 'viewer';
+
+    if (args.screenShareExperience === 'viewer') {
+      const renderElement = document.createElement('img');
+      renderElement.src = 'images/hero.svg';
```
Comment (JamesBurnside) [inline packages/storybook/stories/VideoGallery/VideoGallery.stories.tsx:131]: nit: are there stock images of a windows desktop we could use instead here?

File: packages/storybook/stories/controlsUtils.ts [hunk 2]
```diff
@@ -254,27 +254,11 @@ export const controlsToAdd = {
...
-    name: 'Width (px)'
-  },
-  screenShareSidePanelTileAspectRatio: {
-    control: 'select',
-    options: ['16:9', '3:2', '4:3', '5:9', '1:1'],
-    defaultValue: '16:9',
-    name: 'Side Panel Tile Aspect Ratio (Width:Height)'
-  },
-  screenShareSidePanelWidthRatio: {
+  screenShareExperience: {
```
Comment (JamesBurnside) [inline packages/storybook/stories/controlsUtils.ts:257]: Is there documentation though that tells a developer how their video gallery gets into these state?

File: packages/storybook/stories/VideoGallery/VideoGallery.stories.tsx (hunk 1 above)
Comment (JamesBurnside) [reply packages/storybook/stories/VideoGallery/VideoGallery.stories.tsx:131]: Perhaps: https://microsoft.sharepoint.com/teams/BrandCentral/Pages/Bundles/WIN22_Windows365_Start_V2_3x2_en-US.aspx
(or something here https://microsoft.sharepoint.com/teams/BrandCentral/Search/Pages/BCGeneralResults.aspx?k=windows#anchorScreens)

File: packages/storybook/stories/controlsUtils.ts (hunk 2 above)
Comment (mgamis-msft) [reply packages/storybook/stories/controlsUtils.ts:257]: Not yet. Will add a small section on screensharing in the docs.

**Review by mgamis-msft — APPROVED** [review]

File: packages/storybook/stories/VideoGallery/VideoGallery.stories.tsx (hunk 1 above)
Comment (alcail) [reply packages/storybook/stories/VideoGallery/VideoGallery.stories.tsx:131]: we don't have any in our images folder

File: packages/storybook/stories/controlsUtils.ts (hunk 2 above)
Comment (alcail) [reply packages/storybook/stories/controlsUtils.ts:257]: I don't think so. I'll check and add some if none

File: packages/storybook/stories/VideoGallery/VideoGallery.stories.tsx (hunk 1 above)
Comment (JamesBurnside) [reply packages/storybook/stories/VideoGallery/VideoGallery.stories.tsx:131]: Yeah, you'll need to add one, I assume we are allowed to use the stock images on brand central:
image

cc @ddematheu2

Comment (JamesBurnside) [reply packages/storybook/stories/VideoGallery/VideoGallery.stories.tsx:131]: If not, here is a screencap of my desktop I give permission to use lol
image

File: packages/storybook/stories/controlsUtils.ts (hunk 2 above)
Comment (alcail) [reply packages/storybook/stories/controlsUtils.ts:257]: @JamesBurnside 
adding snippets to story
image
image

File: packages/storybook/stories/VideoGallery/VideoGallery.stories.tsx (hunk 1 above)
Comment (alcail) [reply packages/storybook/stories/VideoGallery/VideoGallery.stories.tsx]: @JamesBurnside updated with 
image

**Pull request merged**